proto/parca/query/v1alpha1/query.proto

```diff
@@ -309,6 +309,9 @@ message Flamegraph {
 
   // untrimmed_total is the total weight of the flame graph before trimming
   int64 untrimmed_total = 9;
+
+  // unfiltered_total is the total weight of the flame graph before trimming
+  int64 unfiltered_total = 10;
```

Comment:
hmm after thinking about this over lunch, should we maybe rather make this part of the overall response, and not the flamegraph itself? this seems like it's not to the specific visualization, but rather the raw data used to create it.

Reply:
That is correct. We want the same for the call graph (see comment above). I mostly added it here because the `untrimmed` was there already. 
Additionally, if we were to add this. I'd be in favor of calling the fields `trimmed` and `filtered` and setting them to the difference. That way the raw flame graph would be `total + trimmed + filtered`. It's really weird to work with right now.

Reply:
Looking at the proto again, would we move all `total` fields of all QueryResponse reports to the top too? 
I feel like this would be a bigger change, including the total field overall too. It might be out of the scope for this PR. However, given we want to be backward compatible, we might not want to merge this PR as is then.

Reply:
Update the proto to the discussed changes.